### DISCUSSION TITLE
Getting help with an open source project for free.

### DISCUSSION THREAD

**Discussion by emullallphotography** [community/community#70029] [Programming Help]
### Body

Hi,

I am a beginner frontend developer moving into backend for a website I am building and I was wondering if I could get extra help with the project, but the problem is I don’t have any money.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by emullallphotography** [comment]
https://github.com/Websitehosterio/websitehosterio.github.io

**Comment by queenofcorgis** [comment]
Per our guidelines:

>❌What is not allowed in this category:
Asking people to code for you or homework/test questions

This type of post is not allowed in our category or community so I'm going to go ahead and close it. 
Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by emullallphotography** [reply]
So, can I still ask questions?

**Reply by queenofcorgis** [reply]
Absolutely 😄 please do! Thanks for asking 👍🏼
